tests/unit/decorators_test.py

```diff
@@ -0,0 +1,104 @@
+import pytest
```

Comment:
Love that it's maintaining 100% coverage!

Reply:
Thanks!  It's fairly straightforward so it wasn't too hard to do though xD